command.go

```diff
@@ -525,14 +564,59 @@ func (c *commandContext) ImportPCD(blob interface{}) error {
 	if err != nil {
 		return err
 	}
-	if err := c.editor.SetPointCloud(p); err != nil {
+	if err := c.editor.SetPointCloud(p, cloudMain); err != nil {
 		return err
 	}
 
 	c.pointCloudUpdated = true
 	return nil
 }
 
+func (c *commandContext) ImportSubPCD(blob interface{}) error {
+	if c.editor.pp == nil {
+		return errors.New("must have base cloud")
+	}
+	p, err := c.pcdIO.importPCD(blob)
+	if err != nil {
+		return err
+	}
+	if err := c.editor.SetPointCloud(p, cloudSub); err != nil {
+		return err
+	}
+
+	c.selectMode = selectModeInsert
+	// Put unit vectors to reconstruct final transformation easily
+	// by cursorsToTrans()
+	c.selected = []mat.Vec3{
+		{},
+		{1, 0, 0},
+		{0, 1, 0},
+		{0, 0, 1},
+	}
+	c.updateRect()
+
+	c.subPointCloudUpdated = true
+	return nil
+}
+
+func (c *commandContext) Do() error {
```

Comment:
[optional] Maybe we could have a more descriptive name for this method.

Reply:
Renamed to `FinalizeCurrentMode`